frontend/src/metabase/visualizations/types/visualization.ts

```diff
@@ -208,7 +214,9 @@ export type VisualizationGridSize = {
 };
 
 // TODO: add component property for the react component instead of the intersection
-export type Visualization = React.ComponentType<VisualizationProps> &
+export type Visualization = React.ComponentType<
+  Omit<VisualizationProps, "width" | "height">
```

Comment:
We should not pass `width` | `height` from `Visualisation.tsx`, it will cause issues.
From what i understand, the `width` and `height` is set by ExplicitSize

Reply:
Outdated